packages/core/test/integration.test.ts

```diff
@@ -25,4 +47,48 @@ describe('integration', () => {
     const doc = await reader.get<{ name: string }>(profileID, writer.id)
     expect(doc).toEqual({ name: 'Alice' })
   })
+
+  test('get an invalid DID should fail', async () => {
+    const invalidDid = 'did:f2_invalid'
+    const reader = new IDX({ ceramic })
+    await expect(reader.get<{ name: string }>('basicProfile', invalidDid)).rejects.toEqual(
```

Comment:
Not an issue but could be more explicit to use `toThrow()` here rather than `toEqual(new Error(...))`